platform/ios/PolyPodApp/PolyPodUITests/SmokeTest.swift

```diff
@@ -0,0 +1,93 @@
+// Please remove this line and the empty one after it
+
+import XCTest
+
+class SmokeTest: XCTestCase {
+    private let app = XCUIApplication()
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        continueAfterFailure = false
+    }
+
+    func testOpenFeaturesNoErrorPopUp() throws {
+        app.launchWithArgs()
+        assertHomeScreenIsShown()
+        
+        let tile_identifiers = ["big_card_view", "medium_card_view", "small_card_view"]
+        
+        for identifier in tile_identifiers {
+            var count = 0
+            var exists = true
+            while exists {
```

Comment:
I _think_ this could be simplified:

```
var count = 0
while featureTileView(index: count++, indentifier: identifier).exists {
    print("\(identifier) - \(count)")
    openFeatureAndCheckForErrorPopup(tile: tile)
}
```

Reply:
Might be cleaner if you keep your loop structure and just get rid of `exists`, doing a `while true` loop, doing a nice old `break` if `tile.exists` is `false`.

Reply:
Yeah, I was contemplating the same thing.